src/main/java/com/khutircraftubackend/jwtToken/JwtUtils.java

```diff
@@ -18,7 +18,7 @@
 public class JwtUtils {
 
     @Value("${jwt.expiration}")
-    private int jwtExpirationSec;
+    private long jwtExpirationSec;
```

Comment:
shall we rename `jwtExpirationSec` into `jwtExpirationMillis` ?

Reply:
Yes, because it is logical when using milliseconds.

Reply:
ok, this still looks the same though, 
hasn't been renamed yet :)